code/modules/research/techweb/all_nodes.dm

```diff
@@ -1260,7 +1260,7 @@
 	id = "adv_mining"
 	display_name = "Advanced Mining Technology"
 	description = "Efficiency Level 127" //dumb mc references
-	prereq_ids = list("basic_mining", "adv_engi", "adv_power", "adv_plasma")
+	prereq_ids = list("basic_mining", "adv_power", "adv_plasma")
```

Comment:
I don't really feel strongly about this, do as you'd like

Reply:
I want this 👍